hr_contract_multi_job/views/hr_contract_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+        <!-- Inherited Contract view -->
```

Comment:
* License should be in every file

Reply:
Hello @alexey-pelykh 

This is new for me, do you know in which Doc I could read about this?

Reply:
Hm, actually, I'm not so sure, yet other reviewers pointed that out. I believe that @pedrobaeza would be much more appropriate to jump in here

Reply:
Well, in XML files there is not the habit of adding the license, but they can be considered code somehow. If you see the template at https://github.com/OCA/maintainer-tools/blob/master/template/module/views/model_name_view.xml#L3, it's included, but I'm not usually forcing this.

Reply:
Right, I'd 👍 to that as would suggest to making it a good tradition